source/agora/node/Validator.d

```diff
@@ -606,8 +606,8 @@ public class Validator : FullNode, API
                                 next_height : enrolled + this.params.ValidatorCycle;
 
         const enrollment = this.enroll_man.createEnrollment(enroll_key, avail_height);
-        log.trace("Sending Enrollment for enrolling at height {} (to validate blocks {} to {})",
-            avail_height, avail_height + 1, avail_height + this.params.ValidatorCycle);
+        log.info("Sending Enrollment with utxo {} for enrolling at height {} (to validate blocks {} to {})",
```

Comment:
The reason it is at trace level currently is because it happens multiple times, not just once.
Essentially every block from X block before the enrollment runs out (where X is configured by the node).

Reply:
Do you think once per block is too much for an important event like sending an enrollment?

Reply:
In fact this is not relevant to this PR anymore so will remove.